test/run-drun/stable-log.mo

```diff
@@ -0,0 +1,73 @@
+import Prim "mo:⛔";
+import StableMemory "stable-mem/StableMemory";
+
+actor {
+
+  func ensure(offset : Nat32) {
+    let pages = (offset + 65536) >> 16;
+      if (pages > StableMemory.size()) {
+        let oldsize = StableMemory.grow(pages - StableMemory.size());
+        assert (oldsize != 0xFFFF);
+     };
+  };
+
+  stable var base : Nat32 = 0;
```

Comment:
It's funny you're storing this one value in a stable var, instead of the first word of stable memory. :)

Reply:
That was deliberate, to show you can use both.